website/server/libs/slack.js

```diff
@@ -239,7 +239,7 @@ function sendShadowMutedPostNotification ({
 function sendSlurNotification ({
   authorEmail,
   author,
-  group,
+  group = undefined,
```

Comment:
It is, but explicitly declaring the parameter this way creates JSDoc type-hinting to IDEs (see attached screenshot) that specify the param is optional (in this case, because a slur report will no longer always relate to a group chat). If this goes against Habitica's coding style guidelines I will happily remove it. 
![sendSlurNotification](https://user-images.githubusercontent.com/12506549/78419949-7f3b8300-7696-11ea-8ce8-cf0a586f92f7.png)

Reply:
sorry for the delay but yeah I would remove this, i understand it might help with some IDEs but I'd prefer not to introduce the change just here